website/docs/r/nrql_alert_condition.html.markdown

```diff
@@ -48,81 +60,166 @@ See additional [examples](#additional-examples).
+<br>For _baseline_ NRQL alert conditions, the value must be within 120-3600 seconds (inclusive).
+<br>For _static_ NRQL alert conditions, the value must be within 120-7200 seconds (inclusive).
 
-- `query` - (Required) The NRQL query to execute for the condition.
-- `since_value` - (Required) The value to be used in the `SINCE <X> MINUTES AGO` clause for the NRQL query. Must be between `1` and `20`.
+- `threshold_occurrences` - (Optional) The criteria for how many data points must be in violation for the specified threshold duration. Valid values are: `all` or `at_least_once` (case insensitive).
+- `duration` - (Optional) **DEPRECATED:** Use `threshold_duration` instead. The duration of time, in _minutes_, that the threshold must violate for in order to create a violation. Must be within 1-120 (inclusive).
+- `time_function` - (Optional) **DEPRECATED:** Use `threshold_occurrences` instead. The criteria for how many data points must be in violation for the specified threshold duration. Valid values are: `all` or `any`.
 
 ## Attributes Reference
 
 In addition to all arguments above, the following attributes are exported:
 
-- `id` - The ID of the NRQL alert condition.
+- `id` - The ID of the NRQL alert condition. This is a composite ID with the format `<policy_id>:<condition_id>` - e.g. `538291:6789035`.
 
 ## Additional Examples
 
+
+##### Type: `baseline`
+
+Example baseline NRQL alert condition for alerting when transaction durations are above a specified threshold and dynamically adjusts based on data trends.
+
+```hcl
+resource "newrelic_alert_policy" "foo" {
+  name = "foo"
+}
+
+resource "newrelic_nrql_alert_condition" "foo" {
+  type                 = "baseline"
+  name                 = "foo"
+  policy_id            = newrelic_alert_policy.foo.id
+  description          = "Alert when transactions are taking too long"
+  enabled              = true
+  runbook_url          = "https://www.example.com"
+  violation_time_limit = "one_hour"
+
+  # baseline type only
+  baseline_direction = "upper_only"
+
+  nrql {
+    query             = "SELECT percentile(duration, 95) FROM Transaction WHERE appName = 'ExampleAppName'"
+    evaluation_offset = 3
+  }
+
+  term {
+    operator              = "above"
+    priority              = "critical"
+    threshold             = 5.5
+    threshold_duration    = 300
+    threshold_occurrences = "all"
+  }
+
+  term {
+    operator              = "above"
+    priority              = "warning"
+    threshold             = 3.5
+    threshold_duration    = 600
+    threshold_occurrences = "all"
+  }
+}
+```
+
 ##### Type: `outlier`
+
+Example outlier detection: Get notified if members of a group deviate by a key metric.
+
+-> **NOTE:** The `outlier` NQRL alert condition type currently does not support new schema attributes introduced in v2.0.0 of the New Relic Terraform provider.
```

Comment:
This is a good example of why I think the new attributes should say something like "introduced in v2" or something similar. 

Either that or we should split off v1 docs into their own section of the docs site as discussed today.

Reply:
I guess in v2.0 we wouldn't include an `outlier` example then?

Reply:
Removed the HCL for the `outlier` example and provided a short message that directly links out to **v1.x** version of the `outlier` example.